### PR TITLE
[NG] Fix tab issue with *ngIf directive

### PR DESCRIPTION
**Pull request by Shijir**
Signed-off-by: Shijir Tsogoo <stsogoo@vmware.com>

### CODE REVIEW

File: src/clarity-angular/layout/tabs/tab-content.ts [hunk 1]
```diff
@@ -14,22 +17,27 @@ let nbTabContentComponents: number = 0;
...
 
     constructor(public ifActiveService: IfActiveService, @Inject(IF_ACTIVE_ID) public id: number,
-                private ariaService: AriaService) {
+                private ariaService: AriaService, private renderer: Renderer2, private el: ElementRef) {
         if (!this.tabContentId) {
             this.tabContentId = "clr-tab-content-" + (nbTabContentComponents++);
         }
+
+        this.setAttributes();
+        this.ifActiveService.currentChange.subscribe(() => {
```
Comment (gnomeontherun) [inline src/clarity-angular/layout/tabs/tab-content.ts]: Looks like you're not adding this to the `subscriptions` array?

File: src/clarity-angular/layout/tabs/tab-link.directive.ts [hunk 2]
```diff
@@ -10,26 +10,26 @@ import {
...
 
 let nbTabLinkComponents: number = 0;
 
 @Directive({
     selector: "[clrTabLink]",
     host: {
-        "[id]": "tabLinkId",
-        "[attr.aria-selected]": "active",
-        "[attr.aria-controls]": "ariaControls",
         "role": "presentation",
```
Comment (gnomeontherun) [inline src/clarity-angular/layout/tabs/tab-link.directive.ts]: If you've replaced some host bindings, would it be worthwhile to replace them all?

File: src/clarity-angular/layout/tabs/tab-content.ts (hunk 1 above)
Comment (Shijir) [reply src/clarity-angular/layout/tabs/tab-content.ts]: Thanks. I have addressed the issue.

File: src/clarity-angular/layout/tabs/tab-link.directive.ts (hunk 2 above)
Comment (Shijir) [reply src/clarity-angular/layout/tabs/tab-link.directive.ts]: I don't see any compelling reasons to replace all of them for now. 🤔  The replaced host bindings were the ones that were holding dynamic values so I needed to use Renderer to set them in a certain lifecycle hook.

**Pull request merged**